core/src/main/kotlin/com/strumenta/kolasu/model/Node.kt

```diff
@@ -2,12 +2,20 @@ package com.strumenta.kolasu.model
 
```

Comment:
Why is `Node` still in the _model_ package and not in the _ast_ package, given that everything else was moved?

Reply:
Originally I put everything in the ast module under the ast package. But now I realize that it does not make sense. In our contest ast and model are synonym, so I will rename the ast package into model